src/core/SimpleLogger.cpp

```diff
@@ -43,7 +43,7 @@ void init_log(std::string filepath, std::string filename) {
     logging::add_common_attributes(); 
     logging::core::get()->add_global_attribute("UTCTime", attrs::utc_clock());
     boost::shared_ptr< file_sink > sink(new file_sink(
-        keywords::file_name = filepath + "/" + filename + ".%Y-%m-%d.%H-%M-%S.%N.log", 
+        keywords::file_name = filepath + "/" + filename + ".%Y-%m-%d.%N.log", 
```

Comment:
Suggest naming this:   filepath + "/ + filename + ".log" because otherwise there will be a LOT of extra log files.   Indicating the shorter name, and the number of files to rotate would be better.  That way we have a small number of log files that are being rotated through, instead of tens (or hundreds) of old files that eventually have to be cleaned up.